playbooks/bootstrap-ssh.yml

```diff
@@ -2,31 +2,27 @@
 - hosts: 127.0.0.1
   connection: local
   gather_facts: false
-  vars:
-    ansible_become: false
```

Comment:
If I run a playbook with `become`, i don't think this will have the intended effect any longer. This playbook gets included in k8s-cluster which is run with `become`.

It might have the same outcome if we set permissions on the generated files/dirs to be owned by the non-root user.